mxcubeweb/core/components/lims.py

```diff
@@ -206,11 +206,30 @@ def select_session(self, session_id: str) -> bool:
                     "[LIMS] Commissioning proposal flag set."
                 )
 
-        HWR.beamline.session.proposal_code = session.code
-        HWR.beamline.session.proposal_number = session.number
-        HWR.beamline.session.session_id = HWR.beamline.lims.get_session_id()
-        HWR.beamline.session.proposal_id = session.proposal_id
-        HWR.beamline.session.set_session_start_date(session.start_date)
+        if HWR.beamline.session.session_id != HWR.beamline.lims.get_session_id():
+            # ruff: noqa: G004
+            logging.getLogger("MX3.HWR").info(
+                f"[LIMS] New session, clearing queue and sample list for {session.code}{session.number}"
+            )
```

Comment:
I think we should probably commit to following what the linter says, even if it is annoying at first. If there is very strong and shared sentiment that we want to go against one particular linter rule then we should disable it globally, so that we have a consistent code base.

As far as I can tell, this would be the preferred way of writing this:

```suggestion
            logging.getLogger("MX3.HWR").info(
                f"[LIMS] New session, clearing queue and sample list for %s%s",
                session.code,
                session.number,
            )
```

If we want to have the policy/guideline that we use f-strings for logging then we should look at the ruff documentation if there is a way to enforce it. I guess it could mean disabling `G004` globally, but that would still mean that there are two ways of doing the same thing %-strings and f-strings. And having two ways is exactly what we should avoid, or at least that is exactly what I want to avoid.